source/code-of-conduct/index.haml

```diff
@@ -0,0 +1,50 @@
+---
+
+%header.header
+  .layer
+    .container.text-center
+      %h1.animated.fadeInUp Code of Conduct of Ruby Belgium
+    %p
+    .container.black
+      .text-columns
+        %p Ruby Belgium is dedicated to providing a harassment-free environment experience for everyone, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof). We want to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible.
+
+        %p We do not tolerate harassment of any associate or full members in any form. Sexual language and imagery is not deemed appropriate for any association activities. Associate and full members violating any of these rules may be at least expelled of the activity. Sanctions may be applied by following the rules defined by the statute of the association.
+
+        %p Harassment includes, but is not limited to: offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion; sexual images in public spaces; deliberate intimidation; stalking; following; harassing photography or recording; sustained disruption of activities; inappropriate physical contact; and any unwelcome sexual attention. Members asked to stop any harassing behaviour are expected to comply immediately.
+
+        %p If a member engages in harassing behaviour, activity organizers or board members may take any action they deem appropriate, including warning the offender or expulsion from the activity. If you are being harassed, notice that someone else is being harassed, or have any other concerns regarding violations of this policy, please contact a member of activity staff or a board member immediately. For each activity, it will be announced how to identify the staff.
+
+        %p Actitivity staff and board members are happy to help participants contact local law enforcement, provide escorts, or in any other way assist those experiencing harassment. Having a safe environment for our members and activity participants is of paramount importance to us, and we value your help towards creating this space.
+
+        %p Beyond creating a safe environment, we want to encourage open citizenship. Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society. If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+        %p Associate and full members, volunteers, and activity participatns are subject to this policy at all association activities and associate-related social events. We invite all our members and those who participate in our activities to help us create safe and positive experiences for everyone.
+
+    %h3.header__subtitle.red.left-spaced
```

Comment:
For sub header keep the consistency with http://rubybelgium.be/rails-girls/ and remove .left-spaced